src/SFA.DAS.Admin.Aan.Web/Controllers/ManageEvent/CheckYourAnswersController.cs

```diff
@@ -62,4 +69,18 @@ private async Task<CheckYourAnswersViewModel> GetViewModel(EventSessionModel ses
 
         return model;
     }
+
+    private Guid GetMemberId()
```

Comment:
Feel free to add this method in the session service as you will need this in many places.